realsense2_camera/config/d455_skynode.yaml

```diff
@@ -0,0 +1,37 @@
+camera1:
+    camera:
+      ros__parameters:
+        device_type: ''
+        serial_no: ''
+        usb_port_id: ''
+
+        config_file: "config/camera-depth-filter-settings.json"
+
+        depth_fps: 10.0
```

Comment:
should we keep it at 15 and let the algorithm decide at what rate to process it?

Reply:
No. Not possible on  the skynode.

Reply:
On USB2 we only have
```
    Depth	  848x480	@ 10Hz	   Z16
    Depth	  848x480	@ 5Hz	   Z16
```